src/main.c

```diff
@@ -74,6 +75,8 @@ static int   g_sudokuCount = 1;				/* in case of -n we can the numbers of sudoku
 static bool  g_outIsPDF;
 static DIFFICULTY g_level = D_EASY;
 static WINDOW *grid, *infobox, *status;
+static move_t	g_undo_stack[UNDO_STACK_SIZE];	/* Stack of previous moves */
+static int   g_undo_stack_ptr = 0;
```

Comment:
Somehow I dont like the `_ptr` without it being a `*`. Maybe `_index` would be a better description? item?

Reply:
makes sense, I like index as well. updated